app-ios/Sources/SponsorFeature/SponsorView.swift

```diff
@@ -21,20 +82,143 @@ public struct SponsorView: View {
+                } else {
+                    ScrollView {
+                        VStack(alignment: .leading, spacing: 24) {
+                            SponsorGridView(
+                                title: "PLATINUM SPONSORS",
+                                sponsors: viewStore.sponsors.filter { $0.plan == .platinum },
+                                columns: 1
+                            )
+                            .onTapItem { sponsor in
+                                viewStore.send(.showLinkSheet(sponsor))
+                            }
+                            Divider().padding(.horizontal, 16)
+                            SponsorGridView(
+                                title: "GOLD SPONSORS",
+                                sponsors: viewStore.sponsors.filter { $0.plan == .gold },
+                                columns: 2
+                            )
+                            .onTapItem { sponsor in
+                                viewStore.send(.showLinkSheet(sponsor))
+                            }
+                            Divider().padding(.horizontal, 16)
+                            SponsorGridView(
+                                title: "SPONSORS",
+                                sponsors: viewStore.sponsors.filter { $0.plan == .supporter },
+                                columns: 2
+                            )
+                            .onTapItem { sponsor in
+                                viewStore.send(.showLinkSheet(sponsor))
+                            }
+                        }
+                        .padding(.horizontal, 16)
+                        .padding(.vertical, 24)
+                    }
+                    .sheet(item: viewStore.binding(get: { $0.sheetItem }, send: .hideLinkSheet)) { item in
+                        if let url = URL(string: item.sponsor.link) {
+                            SafariView(url: url)
+                        }
+                    }
+                }
+            }
+            .task {
+                viewStore.send(.refresh)
+            }
+            .frame(minWidth: 0, maxWidth: .infinity)
+            .frame(minHeight: 0, maxHeight: .infinity)
+            .background(AssetColors.background.swiftUIColor)
+            .foregroundColor(AssetColors.onBackground.swiftUIColor)
+            .navigationTitle(L10n.Sponsor.title)
+            .navigationBarTitleDisplayMode(.inline)
+        }
+    }
+}
+
+struct SponsorGridView: View {
+
+    let title: String
+    let sponsors: [Sponsor]
+    let columns: Int
+    var action: (Sponsor) -> Void = { _ in }
+
+    var body: some View {
+        VStack(alignment: .leading) {
+            Text(title)
+                .font(.system(size: 22, weight: .semibold, design: .default))
+            LazyVGrid(columns: Array(repeating: GridItem(spacing: 16), count: columns), spacing: 16) {
+                ForEach(sponsors, id: \.self) { sponsor in
```

Comment:
How about confirm protocol `Identifiable` to `Sponsor` Model with extension?
if it is, `id: \.self` can be omitted 👍🏼